rust/chroma/src/types.rs

```diff
@@ -1 +1,9 @@
 pub use chroma_api_types::{GetUserIdentityResponse, HeartbeatResponse};
+
+pub use chroma_types::{
+    plan::SearchPayload, AddCollectionRecordsRequest, AddCollectionRecordsResponse, Collection,
+    DeleteCollectionRecordsRequest, DeleteCollectionRecordsResponse, ForkCollectionRequest,
+    GetRequest, GetResponse, IncludeList, InternalSchema, Metadata, QueryRequest, QueryResponse,
+    SearchRequest, SearchResponse, UpdateCollectionRecordsRequest, UpdateCollectionRecordsResponse,
+    UpdateMetadata, UpsertCollectionRecordsRequest, UpsertCollectionRecordsResponse, Where,
+};
```

Comment:
should we just `pub use chroma_types::*`? seems easy to forget to add a type here when modifying the client